uploader/src/main/scala/com/gu/media/UploaderLambda.scala

```diff
@@ -0,0 +1,31 @@
+package com.gu.media
+
+import com.gu.media.UploaderLambda.{Input, Output}
+import com.gu.media.lambda.LambdaWithParams
+import com.gu.media.logging.Logging
+import org.cvogt.play.json.Jsonx
+import play.api.libs.json.Format
+
+class UploaderLambda extends LambdaWithParams[Input, Output] with Logging {
+  override def handleRequest(input: Input): Output = {
+    val youTube = YouTubeClient(config)
+    val output = s"allowedChannels: ${youTube.allowedChannels.mkString(",")}. result: ${input.string * input.integer}"
```

Comment:
Just for testing the skeleton. They will be replaced with the case classes for input/output params (as JSON)